packages/node_modules/@ciscospark/internal-plugin-encryption/test/integration/spec/kms.js

```diff
@@ -91,6 +91,88 @@ describe(`Encryption`, function() {
+        .then(([key]) => spark.internal.encryption.kms.createResource({
+          key
+        }))
+        .then((k) => {
+          kro = k;
+          boundedKeyUri = kro.keyUris[0];
+          assert.lengthOf(kro.authorizationUris, 1);
+        }));
+
+      before(`authorizes a user to a key`, () => spark.internal.encryption.kms.addAuthorization({
+        userIds: [testUserId],
+        kroUri: kro.uri
+      })
+        .then(([auth]) => {
+          assert.equal(auth.resourceUri, kro.uri);
+          assert.equal(auth.authId, testUserId);
+          return assert.isFulfilled(mccoy.spark.internal.encryption.kms.fetchKey({uri: boundedKeyUri}));
+        }));
+
+      before(`authorizes a resource to a key`, () => spark.internal.encryption.kms.createUnboundKeys({count: 1})
+        .then(([key]) => spark.internal.encryption.kms.createResource({key}))
+        .then((k) => {
+          otherKro = k;
+          testResourceId = otherKro.uri;
+          return spark.internal.encryption.kms.addAuthorization({
+            authIds: [testResourceId],
+            kro
+          });
+        })
+        .then(([auth]) => {
+          assert.equal(auth.resourceUri, kro.uri);
+          assert.equal(auth.authId, otherKro.uri);
+          return spark.internal.encryption.kms.listAuthorizations({kroUri: kro.uri})
+            .then((authorizations) => {
+              assert.equal(authorizations.length, 2);
+              assert.include(authorizations, {uri: auth.uri, authId: testUserId, resourceUri: auth.resourceUri});
+              assert.include(authorizations, {uri: auth.uri, authId: testResourceId, resourceUri: auth.resourceUri});
+            });
+        }));
+
+      it(`remove the user and verify this user is not in the authorization list `, () => spark.internal.encryption.kms.removeAuthorization({
+        userId: testUserId,
+        kroUri: kro.uri
+      })
+        .then(([auth]) => {
+          assert.equal(auth.resourceUri, kro.uri);
+          assert.equal(auth.authId, testUserId);
+          return spark.internal.encryption.kms.listAuthorizations({kroUri: kro.uri})
+            .then((authorizations) => {
+              assert.notInclude(authorizations, {uri: auth.uri, authId: testUserId, resourceUri: auth.resourceUri});
+            });
+        }));
+
+
+      it(`remove the resource and verify this resource is not in the authorizaiton list`, () => spark.internal.encryption.kms.removeAuthorization({
+        userId: testResourceId,
+        kroUri: kro.uri
+      })
+        .then(([auth]) => {
+          assert.equal(auth.resourceUri, kro.uri);
+          assert.equal(auth.authId, testResourceId);
+          return spark.internal.encryption.kms.listAuthorizations({kroUri: kro.uri})
+            .then((authorizations) => {
+              assert.notInclude(authorizations, {uri: auth.uri, authId: testResourceId, resourceUri: auth.resourceUri});
```

Comment:
Similar to the statement about deepInclude above, you'll want to map authorizations down to primitives and just compare to testResourceId

Reply:
updated accordingly